README.md

```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
 ### Using møte
 
 Dependencies:
- - Optional: `memcached` (`sudo dnf install memcached`)
- - Python 2.7.x
+ - `memcached` (`sudo dnf install memcached`)
+ - Python 3.8.X
+ - Pipenv
```

Comment:
A link to the Pipenv docs would be a good idea to add here. Also, this is packaged in Fedora too. `sudo dnf install pipenv`

Reply:
Fixed: https://github.com/fedora-infra/mote/pull/141/commits/8bfdd02674b734fa256248bf2429a00fddf28a11